apps/api/src/configuration/index.ts

```diff
@@ -87,35 +71,24 @@ export const Configuration = {
     return env.get('GCP_CDN_URL').asUrlString()
   },
 
-  get cmsPublicUrl() {
-    return this.cmsUrl
-    // return env.get('CMS_PUBLIC_URL').default(this.cmsUrl).asUrlString()
-  },
-
   get cmsAccessToken() {
-    return env
-      .get('CMS_ACCESS_TOKEN')
-      .default('e38bcbf8093f52c19895ef8a4613b48a')
-      .asString()
+    return env.get('CMS_ACCESS_TOKEN').required().asString()
   },
 
   get webUrl() {
     return env.get('WEB_URL').default('http://localhost:3000').asUrlString()
   },
 
   get pinataApiKey() {
-    return env.get('PINATA_API_KEY').default('').asString()
+    return env.get('PINATA_API_KEY').required().asString()
```

Comment:
ah this probably makes debugging missing env vars a lot easier eh? 🙏🏼

Reply:
100%

Though it makes running the build/test workflows a tad more annoying.